enclave/crypto/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+add_library(oecryptombed STATIC
+    ../../common/asn1.c
+    ../../common/cert.c
+    ../../common/kdf.c
+    asn1.c
+    cert.c
+    crl.c
+    ec.c
+    cmac.c
+    hmac.c
+    key.c
+    random.c
```

Comment:
Random.c is in this lib as well as in oeenclave. Is that intentional?

Reply:
Yeah. The random.c in `oeenclave` defines the `oe_rand` function, which basically just calls `oe_random_internal` and the random.c in `oecrypto` has all the mbedtls specific stuff, along with `oe_random_internal`. I'm not sure why there is an `oe_random_internal` in the first space, since all `oe_rand` does is call it.

Reply:
Maybe we could rename one of the files to appropriately reflect the function it defines.

Reply:
I'll name the crypto one `random_internal.c`